contrib/utilities/download_clang_format

```diff
@@ -59,13 +59,25 @@ then
     exit 1
 fi
 
-echo "Downloading and installing clang-format-6."
+echo "Downloading and installing clang-format-6 from ${URL}/${FILENAME}"
 mkdir "${CLANG_PATH}"
 
 tmpdir="${TMPDIR:-/tmp}/dealiiclang${RANDOM}${RANDOM}"
 mkdir -p "${tmpdir}"
 cd "${tmpdir}"
-curl -s -L "${URL}/${FILENAME}" -O > /dev/null
+if [ -x "$(command -v wget)" ]; then
+  echo "Using wget to download..."
+  wget -q --show-progress -L "${URL}/${FILENAME}" > /dev/null
+else
+  if [ -x "$(command -v curl)" ]; then
+    echo "Using wget to download..."
+    curl -q --show-progress -L "${URL}/${FILENAME}" > /dev/null
```

Comment:
Just use the same command for `curl` as originally. 
`curl` uses `-s` for silent mode, doesn't understand `--show-progress` and requires `-O` to write to a local file (instead of `stdout`).